lib/llvmopencl/Flatten.cc

```diff
@@ -74,9 +74,9 @@ static bool flattenAll(Module &M) {
 
   for (llvm::Module::iterator i = M.begin(), e = M.end(); i != e; ++i) {
     llvm::Function *F = &*i;
-    if (F->isDeclaration() || F->getName().starts_with("__pocl_print") ||
-        F->getName() == "__printf_alloc" ||
-        F->getName() == "__printf_flush_buffer" ||
+    if (F->isDeclaration() || F->getName() == "pocl_printf_alloc" ||
+        F->getName() == "pocl_printf_alloc_stub" ||
```

Comment:
Why not add these to AuxFuncs for the targets that require the host-side printf functionality?